Tasks/ServiceFabricUpdateAppVersions/Compare-Xml.psm1

```diff
@@ -0,0 +1,25 @@
+﻿function Compare-Xml
```

Comment:
I decided to rename this to 'Test-XmlEqual' because Compare implies it's returning the differences, when really it's just returning a bool